packages/app-bridge/src/react/useAssetBulkDownload.ts

```diff
@@ -22,7 +22,11 @@ export const useAssetBulkDownload = (appBridge: AppBridgeBlock) => {
 
             const { assetBulkDownloadToken } = await appBridge.api({
                 name: 'getAssetBulkDownloadToken',
-                payload: { settingIds, documentBlockId: appBridge.getBlockId() },
+                payload: {
+                    settingIds,
+                    documentBlockId: appBridge.getBlockId(),
+                    language: appBridge.getTranslationLanguage(),
```

Comment:
Currently, I think we don't even need this PR. Did you check the possibility to just add the language parameter on the web-app side of the app bridge, similar like we did for other similar cases?
<img width="983" alt="image" src="https://github.com/Frontify/brand-sdk/assets/34413733/1e930ea9-6419-4870-a4fc-f4a21d69bdcb">

Probably we need a refactoring to use the `getTranslationLanguage` within the new API structure.